templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+     <li>User: {{ user.get_username }}</li>
```

Comment:
The `user.get_username` should be called with parentheses as `user.get_username()`. This is necessary to correctly invoke the method and display the username.